src/test/java/org/broadinstitute/consent/http/db/DataAccessRequestDAOTest.java

```diff
@@ -129,9 +131,9 @@ void updateNonDraftToDraft() {
     List<DataAccessRequest> draftDars2 = dataAccessRequestDAO.findAllDraftDataAccessRequests();
     assertFalse(draftDars2.isEmpty());
     assertEquals(1, draftDars2.size());
+    assertNotNull(draftDars2.get(0).getDarCode());
```

Comment:
Do we know what the expected value of dar code is here, to use `assertEquals()` instead?

Reply:
Looking at this test, it is very out of date. The method we're using here (`updateDataByReferenceId`) to try and turn a draft into a submitted DAR is not the right method. So, in this test scenario, we do not actually know what the real dar code should be. It's calculated when we create a Dar Collection, which this test method doesn't do.

Reply:
Update to previous comment - we actually are creating a Dar Collection in `createDataAccessRequestV3` so we should know what the dar code is, it should be `dar.getDarCode()`